dev/TeachingTip/TestUI/TeachingTipPage.xaml.cs

```diff
@@ -35,8 +36,17 @@ enum TipLocation
         VisualTree = 0,
         Resources = 1
     }
-    public sealed partial class TeachingTipPage : TestPage
+    public sealed partial class TeachingTipPage : TestPage, INotifyPropertyChanged
     {
+        public event PropertyChangedEventHandler PropertyChanged;
+        private void NotifyPropertyChanged(string propertyName)
```

Comment:
>string [](start = 43, length = 6)

[CallerMemberName] on this would mean the caller doesn't need to specify the nameof() parameter, it'll just be inferred correctly.

Reply:
Ah, that was the thing I couldn't remember. I was looking for it. :)